addons/source-python/packages/source-python/listeners/tick.py

```diff
@@ -109,19 +109,17 @@ def add(self, delay):
 class Delay(WeakAutoUnload):
     """Execute a callback after a given delay."""
 
-    def __init__(
-        self, delay, callback, cancel_on_map_end=False, *args, **kwargs
-    ):
+    def __init__(self, delay, callback, cancel_on_map_end=False, args=(), kwargs={}):
```

Comment:
I have already made most of the changes and pushed them, though I did miss the doc strings.  However, you should never declare default values for arguments as mutable types.  I guess the tuple one is fine, but I used None in my implementation.  But a dictionary should never be used, since it can be updated and cause issues.